app/views/idv/doc_auth/upload.html.erb

```diff
@@ -37,6 +37,7 @@
       <%= render PhoneInputComponent.new(
             form: f,
             required: true,
+            hint: t('forms.example') + ' (201) 555-0123',
```

Comment:
Prior to #7917, the example updated when the country was changed. Would it be expected to do so here?